### PR TITLE
Add Knapsack exercise

### PR DESCRIPTION
**Pull request by kahgoh**
This is one of the [48in24 exercises](https://forum.exercism.org/t/48in24-exercise-08-13-knapsack/12054).

### CODE REVIEW

**Comment by kahgoh** [comment]
@ErikSchierboom this one is ready for review

File: config.json [hunk 1]
```diff
@@ -618,16 +591,18 @@
...
+        "practices": [],
+        "prerequisites": [],
+        "difficulty": 5
       }
     ]
   },
-  "foregone": [
-    "bank-account",
-    "parallel-letter-frequency"
-  ],
```
Comment (ErikSchierboom) [inline config.json:629]: This key is now gone, but it should be moved under the `exercises` key.

File: config.json [hunk 2]
```diff
@@ -618,16 +591,22 @@
...
+        "practices": [],
+        "prerequisites": [],
+        "difficulty": 5
       }
     ]
   },
   "foregone": [
     "bank-account",
     "parallel-letter-frequency"
   ],
```
Comment (ErikSchierboom) [inline config.json:609]: Thanks for undoing the remove, but this is still at the root key, instead of within the "exercises" key.

Comment (kahgoh) [reply config.json]: Apologies, I misunderstood the comment earlier

**Review by ErikSchierboom — APPROVED** [review]

**Pull request merged**